code/modules/mob/mob.dm

```diff
@@ -1098,6 +1098,10 @@
 /mob/proc/has_nightvision()
 	return see_in_dark >= NIGHTVISION_FOV_RANGE
 
+/// Is this mob affected by nearsight
+/mob/proc/is_nearsighted()
+	return FALSE
```

Comment:
The base level of `is_nearsighted()` should check for `HAS_TRAIT(src, TRAIT_NEARSIGHT)`. The nearsight trait isn't human exclusive (`become_nearsighted()` is a living level proc). 

Then in the human level proc just check for the paren't return value as well with the ol'
```
. = ..()
if(!.)
   return FALSE
```

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/66327/commits/58573b68f931a7678463ff2f797c86fb23c8f2d2 and  https://github.com/tgstation/tgstation/pull/66327/commits/67791dfff6ff35470e3793780e2ef172d968d037